microcosm_postgres/example.py

```diff
@@ -29,6 +29,7 @@ class Employee(EntityMixin, Model):
 
     first = Column(String(255), nullable=False)
     last = Column(String(255), nullable=False)
+    other = Column(String(255), nullable=True)
```

Comment:
The repo includes a few example models (which are used in the tests). I opted to keep these in the source tree (vs the test tree) so that we can layer other components (e.g. REST conventions) on top of this library and not rewrite the models.

In this case, I introduced a trivial schema change to test Alembic migrations. It has no effect on the tests.